homeassistant/components/tuya/select.py

```diff
@@ -27,6 +27,28 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Motion Sensor Light
+    # https://developer.tuya.com/en/docs/iot/gyd?id=Kaof8a8hycfmy
+    "gyd": (
+        SelectEntityDescription(
+            key=DPCode.DEVICE_MODE,
```

Comment:
Please provide translation functionality for these values, by defining a custom device class and add the translations to the strings file.

Reply:
Translation facility added